int64 tick_id
copy_when_uncertain/BeeUpdate[] bee_updates

---

int64 tick_id
